Files/Interacts/Interaction.cs

```diff
@@ -535,15 +535,26 @@ private async Task OpenPropertiesWindow(object item)
         {
             if (ApiInformation.IsApiContractPresent("Windows.Foundation.UniversalApiContract", 8))
             {
+
                 AppWindow appWindow = await AppWindow.TryCreateAsync();
                 Frame frame = new Frame();
+                frame.KeyDown += Frame_KeyDown;
                 appWindow.TitleBar.ExtendsContentIntoTitleBar = true;
                 frame.Navigate(typeof(Properties), item, new SuppressNavigationTransitionInfo());
+                
                 WindowManagementPreview.SetPreferredMinSize(appWindow, new Size(400, 500));
 
                 appWindow.RequestSize(new Size(400, 475));
                 appWindow.Title = ResourceController.GetTranslation("PropertiesTitle");
 
+                async void Frame_KeyDown(object sender, KeyRoutedEventArgs e)
```

Comment:
I think it is better to do it inside `Properties.xaml.cs` as it is n't a part of interaction.